frontend/src/ContentArea/ContentArea.js

```diff
@@ -1,43 +1,104 @@
 import React from 'react';
-import JSONToHTML from '../JSONHandle';
+import { Redirect } from 'react-router';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import marked from 'marked';
 
 export default class ContentArea extends React.Component {
-  constructor(props) {
-    super(props);
+  componentWillMount() {}
 
-    this.state = {
-      innerHTML: undefined,
-    };
+  render() {
```

Comment:
maybe refactor the viewy bits out to components?

Reply:
could not understand what you mean. type please which line needs refactor.

Reply:
not "line", *function*. the render function is a bit long and hard to understand. Factoring out the view bits as components would help I think.